### DISCUSSION TITLE
Q1

### DISCUSSION THREAD

**Discussion by Innocentsax** [Innocentsax/DSA_IN_JAVA#12] [Q&A] [answered]
How do Data Structure and System Design helps in building Business logic

**Comment by SamuelOlawuyi** [comment, marked as answer]
Data structure and system design solve complex problems by using intuitive methodology, this also can be integrated into business to build a solid and effective system.

**Comment by Chrysz-Ojo** [comment]
Data structure and system design are  way of organizing and storing data in a computer's memory so that it can be efficiently accessed and manipulated. Different data structures are designed to suit various types of applications and operations, such as searching, inserting, deleting, and sorting data.
